src/modules/commander/Arming/PreFlightCheck/checks/powerCheck.cpp

```diff
@@ -63,6 +63,11 @@ bool PreFlightCheck::powerCheck(orb_advert_t *mavlink_log_pub, const vehicle_sta
 		return true;
 	}
 
+	if (status.hil_state == vehicle_status_s::HIL_STATE_ON) {
```

Comment:
I would expect that eventually we would instead want the `system_power` also bring published from the simulated battery model instead of using this as a workaround?

Reply:
We could, sure.